src/Bundle/Resources/skeleton/Factory.tpl.php

```diff
@@ -39,6 +39,11 @@ protected function getDefaults(): array
     {
         return [
             // TODO add your default values here (https://github.com/zenstruck/foundry#model-factories)
+<?php
+foreach ($defaultProperties as $fieldname => $type) {
+        echo "            '".$fieldname."' => ".$type.",\n";
```

Comment:
As it stands now, the todo comments will generate invalid php syntax:

```php
return [
    'field1' => self::faker()->text(),
    'field2' => // TODO... (missing a value and comma)
    'field3' => self::faker()->text(),
];
```

I'm thinking the value returned from `Property::createFakerMethodFromDoctrineType()` should include the comma and the todo value should be `null` (?). The above would look like this:

```php
return [
    'field1' => self::faker()->text(),
    'field2' => null, // TODO...
    'field3' => self::faker()->text(),
];
```